src/bin/bench.rs

```diff
@@ -0,0 +1,503 @@
+extern crate clap;
+extern crate num_cpus;
+#[macro_use]
+extern crate log;
+extern crate rsdb;
+
+use std::error::Error;
+use std::io::prelude::*;
+use std::process;
+use std::collections::HashMap;
+
+use clap::{App, Arg};
+
+fn main() {
+    let cpus = &(num_cpus::get().to_string());
+
+    let matches = App::new("RSDB bench")
+        .version("0.1.0")
+        .about("RSDB benchmarking tool")
+        .author("Tyler Neely <t@jujit.su>")
```

Comment:
😅 thanks (don't wanted to be rude here)!

Just pushed a change with an update! 👍